src/zmq.cpp

```diff
@@ -688,12 +688,12 @@ const char *zmq_msg_gets (const zmq_msg_t *msg_, const char *property_)
     // Polling.
 
 #if defined ZMQ_HAVE_POLLER
-inline int zmq_poller_poll (zmq_pollitem_t *items_, int nitems_, long timeout_)
+int zmq_poller_poll (zmq_pollitem_t *items_, int nitems_, long timeout_)
 {
     // implement zmq_poll on top of zmq_poller
     int rc;
     zmq_poller_event_t *events;
-    zmq::socket_poller_t poller;
+    zmq::socket_poller_t *poller = new zmq::socket_poller_t;
```

Comment:
IIRC this was on the stack intentionally, as it will murder performances otherwise

Reply:
This is crashing on my Windows computer otherwise (stack overflow)

Reply:
This is documented in the INSTALL file, you need to increase the stack size: https://github.com/zeromq/libzmq/blob/master/INSTALL#L49